go/chat/attachments/s3.go

```diff
@@ -87,7 +87,13 @@ func (a *S3Store) PutS3(ctx context.Context, r io.Reader, size int64, task *Uplo
 	region := a.regionFromParams(task.S3Params)
 	b := a.s3Conn(task.S3Signer, region, task.S3Params.AccessKey).Bucket(task.S3Params.Bucket)
 
-	if size <= minMultiSize {
+	multiPartUpload := size <= minMultiSize
+	if multiPartUpload && a.env.GetAttachmentDisableMulti() {
```

Comment:
`||`?

Reply:
I think `&&` is correct. I wanted to only print that line if we turn that variable off because of the config. So if `multiPartUpload` is true, and we disabled it, then it logs that line and sets it to false.

Reply:
ah gotcha i guess because the condition was flipped i was thinking we wouldn't properly set `multiPartUpload = false` but the fix makes it make sense